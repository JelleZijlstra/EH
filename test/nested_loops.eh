for m in [1, 2]
    for n in [3, 4]
        printvar(m, n)
    end
end
