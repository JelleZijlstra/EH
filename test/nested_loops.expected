%%stdout%%
@tuple <2> [
	@int 1
	@int 3
]
@tuple <2> [
	@int 1
	@int 4
]
@tuple <2> [
	@int 2
	@int 3
]
@tuple <2> [
	@int 2
	@int 4
]
%%stderr%%
